apps/AEPSampleAppNewArchEnabled/ios/AEPSampleAppNewArchEnabled/AppDelegate.mm

```diff
@@ -0,0 +1,66 @@
+#import "AppDelegate.h"
+#import "AdobeBridge.h"
+
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTLinkingManager.h>
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  self.moduleName = @"main";
+
+  // You can add your custom initial props in the dictionary below.
+  // They will be passed down to the ViewController used by React Native.
+  self.initialProps = @{};
+
+  [AdobeBridge configure: application.applicationState];
```

Comment:
We're missing Lifecycle implementation in this file, see public docs for more details: https://developer.adobe.com/client-sdks/home/base/mobile-core/lifecycle/#register-lifecycle-with-mobile-core-and-add-appropriate-startpause-calls

Reply:
need to discuss this with you on a call

Reply:
done

Reply:
have added the code in adobeBridge.m as the registration code for extension was already been there